Brain_I2C/Brain_I2C.ino

```diff
@@ -45,10 +50,50 @@ void loop() {
       index++;
     }
     for (int i=0; i<6; i++) {
-      Serial.print(farrbconvert.sensorReadings[i]);
-      Serial.print(" ");
+      float reading = farrbconvert.sensorReadings[i];
+      if (reading > 0) {
+      }
+    }
+    String packet = make_packet(sensor);
+    Serial.println(packet);
+    RFSerial.println(packet);
+    delay(sensor.clock_freq);
+  }
+}
+
+String make_packet(struct sensorInfo sensor) {
+  String packet = "{";
+  packet += (String)sensor.overall_id;
+  packet += ",";
+  char *data = 0;
```

Comment:
Why do I have to specify it? You don't need to do that, it still compiles

Reply:
because you won't necessarily get all the memory you need. If this array needs to have 100 elements, but you define it like this, you will only have access to whatever memory is currently available. You'll need to specify the size of the array in order guarantee that you get enough space there.